test/test_typed_array.cpp

```diff
@@ -520,20 +513,4 @@ TEST_SUITE("typed_array")
         float64_t
     );
 
-    TEST_CASE_TEMPLATE_INVOKE(
```

Comment:
Just to clarify, this was removed because it's duplicating the previous block?

Reply:
The difference isthe std::string entry

Reply:
We used to run the "constructor from n-times same value" for all types **except string**
I removed this limitation, therefore we need not extra `TEST_CASE_TEMPLATE_INVOKE` and can just put the "constructor from n-times same value" in the pre-existing invocation